docs/FAQ.md

```diff
@@ -40,13 +40,18 @@ The Yield Protocol v1 only permits borrowing and lending of Dai, and only accept
 Yield is not built with UMA. It is built on top of Maker.
 
 ### Are there any major differences between fyDai and UMA's uUSD?
-The main differences are the collateral assets, the settlement assets, and the pool mechanics.
```

Comment:
I think we should remove the pool mechanics since UMA could use YieldSpace, in theory. Suggested answer: 
 
One significant difference is that UMA's uUSD is cash-settled while Yield's fyDai is physically settled. Specifically, when UMA's uUSD expires, the outstanding uUSD becomes "de-pegged", and redeemable for the underlying collateral: WETH for uUSDwETH, renBTC for uUSDrBTC. Conversely, when Yield Protocol's fyDai expires it is redeemable for Dai, so the fixed rate becomes a variable rate (the DSR). This could be seen as "opening a Maker Vault on the borrower's behalf."

Reply:
UMA could use YieldSpace, but it does not. Right now is something in which Yield is better than UMA for fixed-rate, and we should clearly state so.